packages/data-explorer-ui/src/components/Error/error.tsx

```diff
@@ -35,6 +67,21 @@ export const Error = ({ rootPath }: ErrorProps): JSX.Element => {
             </Link>
           </SectionActions>
         )}
+        {(requestUrlMessage || errorMessage) && (
```

Comment:
Move the `Divider` and each `ErrorDetailSection` below the closing of `ErrorSection`